be/.env.example

```diff
@@ -0,0 +1,4 @@
+# Connection string for the db
+DATABASE_URL="postgresql://username:password@localhost:PORT/databasename"
```

Comment:
> 
> 
> > > > Because i'm not using nestjs
> > > 
> > > 
> > > Makes sense. Any particular reason against nest? The things I currently like about it are
> > > ```
> > > * It's popular and well documented
> > > 
> > > * Supports ts out of the box
> > > 
> > > * Supports apollo and graphql out of the box
> > > 
> > > * Testing is super easy with it
> > > 
> > > * It's super opinionated (meaning standards and good practices are set for us and we don't have to invent them our-self)
> > > ```
> > 
> > 
> > I haven't learned it and I need to work on my School End Project where I'm using node for backend.
> 
> nest is node backend

if you compare it to my code it's different

Reply:
address `maybe split this to separate arguments so username, password, host and dbname are different variables. It would then be more clear what changed if in future any of them are changed. Also then they can be individually overriden by system env or even in tests.``